components/header/README.md

```diff
@@ -0,0 +1,123 @@
+---
+description: Headers help users understand what the content of the page is about and provides a quick, organized way to reach the main sections of a website.
+---
+# Header
+
+Headers help users understand what the content of the page is about and provide a quick, organized way to reach the main sections of a website. They appear at the top of a page, above the main body text.
+
+### Required
+### Last Updated: 2 days ago
```

Comment:
Last updated should probably be a definitive date instead of something relative otherwise no matter what the user would perceive this to have been updated recently.